ffi/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ffi"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
same

Reply:
Revert to `0.4` in commit [61c6f1b](https://github.com/scroll-tech/scroll-prover/pull/178/commits/61c6f1bced2969240af198db8e502c48d9f5c618).